scripts/watch_containers.py

```diff
@@ -0,0 +1,63 @@
+import docker
+import time
+import argparse
+from datetime import datetime
+from datetime import timedelta
+
+
+def watch_containers(name, since=datetime.now(), pooling_time=60, notification_channel='stdout'):
+    api_client = docker.APIClient(base_url='unix://var/run/docker.sock')
+    client = docker.from_env()
+    containers = get_active_containers(client, api_client, name, since)
+    successful_containers = []
+    failed_containers = []
+    while(True):
+        active_containers = get_active_containers(client, api_client, name, since)
+        if not active_containers:
+            print("No more active containers. Exiting...")
+            break
+        else:
+            print("Still running containers. Sleeping...")
+            time.sleep(pooling_time)
+
+    for container in containers:
+        exit_code = api_client.inspect_container(container.id)['State']['ExitCode']
```

Comment:
If the container is stopped and removed (for example doing `docker-compose down`) this will fail. We could add a new list `killed_containers` and add these containers there to show them on the stats too

Reply:
``` ❯ python3 scripts/watch_containers.py bigmetadata_bigmetadata_run_ --since '2018-03-02 16:28:00'       [17:30:48]
Still running containers. Sleeping...
No more active containers. Exiting...
Finished. Total 1 -- Successful 0 -- Failed 0 -- Deleted 1. It tooks 3 minutes
>>> elapsed time 1m1s   ```